docs/source/installation-guide.rst

```diff
@@ -11,3 +11,4 @@ running on your own infrastructure.
    quickstart
    quickstart-docker
    installation-basics
+   installation-guide-hard
```

Comment:
I believe the test is failing because there's a blank line missing at the end of this file.